src/utils/whitelistApis.ts

```diff
@@ -3013,6 +3013,13 @@ export const API_LIST = {
                 ROLE.SPV_ADMIN,
             ],
         },
+        '/proxies/v8/workflow/v2/userWFApplicationFieldsSearch': {
```

Comment:
KONG API points to workflowhanlder service prefix... which means you API should be 
/proxies/v8/workflowhandler/v2/userWFApplicationFieldsSearch

this requires new handler in proxies_v8.ts file too

Reply:
Done